CacheMoney/frontend/src/components/RegisterView.js

```diff
@@ -7,14 +7,16 @@ import Toggle from "./style/Toggle";
 import { lightTheme, darkTheme } from "../components/style/Themes";
 import { useNavigate } from "react-router-dom";
 import config from "../config.js";
-import { ToastContainer, toast } from 'react-toastify';
-import 'react-toastify/dist/ReactToastify.css';
+
```

Comment:
This file looks like it's overwriting email team's changes. change it back across the file